scilpy/connectivity/connectivity.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+import copy
+import logging
+import os
+import sys
+import threading
+
+from dipy.io.utils import is_header_compatible, get_reference_info
+from dipy.tracking.streamlinespeed import length
+from dipy.tracking.vox2track import _streamlines_in_mask
+import h5py
+import nibabel as nib
+import numpy as np
+
+from scilpy.image.labels import get_data_as_labels
+from scilpy.io.hdf5 import (assert_header_compatible_hdf5,
+                            reconstruct_streamlines_from_hdf5)
+from scilpy.tractanalysis.reproducibility_measures import \
+    compute_bundle_adjacency_voxel
+from scilpy.tractanalysis.streamlines_metrics import compute_tract_counts_map
+from scilpy.utils.metrics_tools import compute_lesion_stats
+
+d = threading.local()
+
+
+def load_node_nifti(directory, in_label, out_label, ref_img):
+    in_filename = os.path.join(directory,
+                               '{}_{}.nii.gz'.format(in_label, out_label))
+
+    if os.path.isfile(in_filename):
+        if not is_header_compatible(in_filename, ref_img):
+            raise IOError('{} do not have a compatible header'.format(
+                in_filename))
+        return nib.load(in_filename).get_fdata(dtype=np.float64)
```

Comment:
float32 enough ?

Reply:
@frheault ? I changed it to 32 as per this suggestion. Is it ok?